utils/mediaupload.js

```diff
@@ -98,16 +98,22 @@ export function mediaUpload( {
 				};
 				setAndUpdateFiles( idx, mediaObject );
 			},
-			() => {
+			( response ) => {
 				// Reset to empty on failure.
 				setAndUpdateFiles( idx, null );
-				onError( {
-					code: 'GENERAL',
-					message: sprintf(
+				let message;
+				if ( ! isEmpty( response.responseJSON.message ) ) {
```

Comment:
It might be safer to use `has` or `get` from lodash in here in case `responseJSON` is not set:
```js
if ( has( response, [ 'responseJSON', 'message' ] ) {
```

Reply:
Good suggestion. Updated in 20d673af4